lib/v2/v2rayshare/index.js

```diff
@@ -0,0 +1,27 @@
+const got = require('@/utils/got'); // 自订的 got
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { data: response } = await got('https://v2rayshare.com/wp-json/wp/v2/posts/?per_page=10');
+
+    const items = [];
+    for (const responseKey in response) {
+        const data = response[responseKey];
+        const a = {
+            title: data.title.rendered,
+            link: data.link,
+            author: 'V2rayShare',
+            category: ['免费节点'],
+            pubDate: parseDate(data.date),
+            description: data.content.rendered,
+        };
+        items.push(a);
+    }
```

Comment:
```suggestion
    const items = response.map((item) => ({
        title: item.title.rendered,
        link: item.link,
        author: 'V2rayShare',
        category: ['免费节点'],
        pubDate: parseDate(item.date_gmt),
        description: item.content.rendered,
    }));
```
You can use [.map()](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create a new array.

Reply:
非常感谢~